src/components/studentListPage/FilterBar/FilterBar.jsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import "bootstrap/dist/css/bootstrap.min.css";
+import "font-awesome/css/font-awesome.min.css";
+import "./FilterBarStyles.css";
+
+class FilterBar extends React.Component {
+    render() {
+        return (
+            <div className="student-page-filter-bar">
+                <div className="filter-bar-college btn-group dropup">
+                    <span className="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                        <span>
+                            <i className="fa fa-filter" aria-hidden="true"></i> Branch
+                        </span>
+                    </span>
+                    <ul className="dropdown-menu">
+                        <li>
+                            <a href="#" data-value="option1" tabIndex="-1">
```

Comment:
the checkbox should be checked when clicked anywhere on the label. Do not forward href to "#". Do it everywhere.